Pod/Classes/WPMediaPickerHelper.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface WPMediaPickerHelper : NSObject
```

Comment:
Great to pull this code out, but I wonder if this class could have a name more closely related to what it does? `Helper` is very generic 🙂

Reply:
That's done! 🙂